python/examples/science_benchcmarks/uno/data_processing.py

```diff
@@ -0,0 +1,156 @@
+import os
+import time
+import download_util
+from pycylon import CylonContext
+from pycylon import Table
+from pycylon.io import CSVReadOptions
+from pycylon.io import read_csv
+import numpy as np
+import pandas as pd
+
+ctx: CylonContext = CylonContext(config=None, distributed=False)
+
+
+def load_aggregated_single_response_pandas(target='AUC', min_r2_fit=0.3, max_ec50_se=3.0,
+                                           combo_format=True,
+                                           rename=False):
+    url = "https://ftp.mcs.anl.gov/pub/candle/public/benchmarks/Pilot1/combo/combined_single_response_agg"
+    output_combined_single_response = \
+        "/home/vibhatha/data/uno/Pilot1/workload_1/combined_single_response_agg"
+
+    if not os.path.exists(output_combined_single_response):
```

Comment:
Shall we move these to another repository?

Reply:
Sure let’s do that